vertex2f[] m_points
